bouncer/views.py

```diff
@@ -32,7 +32,7 @@ def annotation(self):
         settings = self.request.registry.settings
 
         try:
-            document = util.elasticsearch_client(settings).get(
+            document = self.request.es.get(
```

Comment:
`bouncer.util.elasticsearch_client` is not used anymore now and should be deleted.